indices/get_mesh_headings.py

```diff
@@ -118,36 +131,35 @@ def get_headings(current_node: Node) -> List[str]:
     parser.add_argument('out_file', help='Location to store the headings to use')
     args = parser.parse_args()
 
-    descriptors_by_depth = {}
+    descriptors_by_depth = defaultdict(list)
 
     with open(args.mesh_file) as in_file:
-
         heading = None
-        ids = None
+        ids = []
         for line in in_file:
             line = line.strip()
             if 'NEWRECORD' in line:
-                if ids is not None:
-                    for id in ids:
-                        depth = len(id.split('.'))
-
-                        if depth in descriptors_by_depth:
-                            descriptors_by_depth[depth].append((heading, id))
-                        else:
-                            descriptors_by_depth[depth] = [(heading, id)]
+                for id in ids:
+                    # H01 is the code for Natural Science Disciplines,
+                    # L01 is the code for information science
+                    # TODO consider adding Humanities (k01) as well
+                    if 'H01' not in id:
+                        continue
+                    depth = len(id.split('.'))
+
+                    if depth in descriptors_by_depth:
+                        descriptors_by_depth[depth].append((heading, id))
+                    else:
+                        descriptors_by_depth[depth] = [(heading, id)]
```

Comment:
Isn't the point of `defaultdict` to not need to check if the key is in the dict? So I think you could just do this:
```suggestion
                    descriptors_by_depth[depth].append((heading, id))
```

Reply:
:facepalm: yeah I added the change specifically to get rid of those lines, then forgot to get rid of those lines